new.go

```diff
@@ -113,6 +116,15 @@ func newBuilder(ctx context.Context, store storage.Store, options BuilderOptions
 		options.FromImage = ""
 	}
 
+	if options.NetworkInterface == nil {
+		// create the network interface
+		// Note: It is important to do this before we pull any images/create containers.
```

Comment:
Where does someone like me who's reading this comment go to learn why this is important?

Reply:
The default network backend detection logic needs an empty store to correctly detect that we can use netavark, if the store was not empty it will use CNI to not break existing installs.
If we do this after the pull it will always use CNI even when we would like to use netavark, e.g new installs. 

I will add this to the comment